rules-tests/Transform/Rector/Attribute/AttributeKeyToClassConstFetchRector/Fixture/some_class.php.inc

```diff
@@ -2,12 +2,12 @@
 
 namespace Rector\Tests\Transform\Rector\Attribute\AttributeKeyToClassConstFetchRector\Fixture;
 
-use Doctrine\ORM\Mapping\Column;
+use Rector\Tests\Transform\Rector\Attribute\AttributeKeyToClassConstFetchRector\Source\TestAttribute;
 
 class SomeClass
 {
-    #[Column(type: "string")]
-    public $name;
+    #[TestAttribute(type: "value")]
```

Comment:
I changed this to the TestAttribute I added.
The behavior can be different when working with not existing classes.

Reply:
What is the different?, if you got the `FullyQualified` of class constant fetch, compare `__toString()` should be equal

Reply:
It wouldn't have been necessary for the attribute, but for the constant the behavior definetly changed, because the resolved value changed. I just wanted to clean up the existing test a little, to work with existing classes.

Reply:
The generated value should not change even class not exists (not autoloaded), as it uses `ClassConstFetchNameResolver`

Could you rollback existing test, and add  skipped test as:

```php
use Doctrine\ORM\Mapping\Column;

use Doctrine\DBAL\Types\Types;

class SkipAlready
{
    #[Column(type: Types::STRING)]
    public $name;
}
```

Reply:
Done